samples/Samples.Amazon.SQS/Samples.Amazon.SQS.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">3.3.3</ApiVersion>
+    <DefineConstants Condition="'$(ApiVersion)'>='3.0.0'">$(DefineConstants);SQS_3_0</DefineConstants>
```

Comment:
Was there any breaking API change that required a separate `SQS_3_0` compilation flag?

Reply:
Ah, nope, I was just playing.